resources/tag.py

```diff
@@ -132,31 +134,46 @@ def put(self, type, identifier):
               schema:
                 type: string
               description: Object identifier
+        requestBody:
+            description: Tag value
+            content:
+              application/json:
+                schema: TagRequestSchema
         responses:
             200:
                 description: When tag is successfully added
+                content:
+                  application/json:
+                    schema:
+                      type: array
+                      items:
+                        $ref: '#/components/schemas/TagItemResponse'
             400:
-                description: When tag or request body isn't valid
+                description: When tag isn't valid
             403:
                 description: When user doesn't have `adding_tags` capability.
             404:
                 description: When object doesn't exist or user doesn't have access to this object.
         """
-        schema = TagSchema()
+        schema = TagRequestSchema()
         obj = schema.loads(request.get_data(as_text=True))
-
         if obj.errors:
             return {"errors": obj.errors}, 400
 
         db_object = access_object(type, identifier)
         if db_object is None:
             raise NotFound("Object not found")
-        tag_name = obj.data["tag"].lower().strip()
+        tag_name = obj.data["tag"]
 
-        was_modified = db_object.add_tag(tag_name)
+        db_object.add_tag(tag_name)
```

Comment:
```suggestion

        tag_name = obj.data["tag"]
        db_object.add_tag(tag_name)
```
stylistic change, but implies that `tag_name` is used in the 2nd block, not first